build.gradle

```diff
@@ -55,14 +55,21 @@ jenkinsPlugin {
 sourceCompatibility = '1.7'
 
 dependencies {
-  compile 'org.jenkins-ci.lib:dry-run-lib:0.1'
-  compileOnly 'org.jenkins-ci:symbol-annotation:1.3'
-  jenkinsPlugins 'org.jenkins-ci.plugins:structs:1.3@jar'
+  jenkinsPlugins 'org.jenkins-ci.plugins:dry-run:0.9'
```

Comment:
The new version of the `dry-run` plugin seems to depend on the `maven-plugin`. Can we exclude the `maven-plugin` as a dependency? Something like
```
configurations {
  all {
    exclude group: 'org.jenkins-ci.main', module: 'maven-plugin'
  }
}
```